src/Ganeti/HTools/Loader.hs

```diff
@@ -372,6 +373,14 @@ clearDynU cdata@(ClusterData _ _ il _ _) =
   let il2 = Container.map (\ inst -> inst {Instance.util = zeroUtil }) il
   in Ok cdata { cdInstances = il2 }
 
+-- | Update cluster data to use static node memory on KVM.
+setStaticKvmNodeMem :: ClusterData -> Int -> Result ClusterData
+setStaticKvmNodeMem cdata@(ClusterData _ nl _ _ _) staticNodeMem =
```

Comment:
I've seen nl being used for node lists (same as il for instance list) quite frequently in the codebase. I agree that it isn't very readable though. Also the usage of the apostrophe is overloaded, it is frequently used to denote a mutated version of the original variable as well as the strict version of a function as you mentioned earlier.